src/modules/products/components/ProductCard.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Product } from "../interfaces/product.interface.ts";
+import { Link } from "react-router-dom";
+
+interface ProductCardP {
```

Comment:
Estaría mejor un nombre mas descriptivo. Algo como:
```suggestion
interface ProductCardProps {
```